controls/nist_ocp4.yml

```diff
@@ -12547,37 +12552,59 @@ controls:
     and incorporated into SA-20.
   rules: []
 - id: SA-15
-  status: pending
+  status: inherently met
   notes: |-
-    Section a: A control response to SA-15(a) is planned. Progress
+    Section a.1: A control response to SA-15(a)(1) is planned. Progress
     can be tracked at:
 
-    https://issues.redhat.com/browse/CMP-383
+    Security has continuous and integrated at every stage of the app and
+    infrastructure life cycle for OpenShift.[1]
+   
+    Section a.2:
+    DevOps security practices are adapt to the OpenShift development and align
```

Comment:
Sorry, I'm not sure what is this point saying? Is the OpenShift development aligned with those guidelines?

Reply:
I kinda feel Section a.2: and Section a.3 are the same, wondering if we can use the same response for it.